store/src/main/java/com/onlineStore/coherent/RandomStorePopulator.java

```diff
@@ -2,53 +2,44 @@
-            new Product(faker.beer().name(),  faker.number().numberBetween(1, 10), faker.number().numberBetween(100, 1500))
-    ));
-
-    //Category Book
-    categoryMap.put(new Category("Book"), Arrays.asList(
-            new Product(faker.book().title(),  faker.number().numberBetween(1, 10), faker.number().numberBetween(100, 1500)),
-            new Product(faker.book().title(),  faker.number().numberBetween(1, 10), faker.number().numberBetween(100, 1500)),
-            new Product(faker.book().title(),  faker.number().numberBetween(1, 10), faker.number().numberBetween(100, 1500))
-    ));
-
-    //Category Medicine
-    categoryMap.put(new Category("Medicine"), Arrays.asList(
-            new Product(faker.medical().medicineName(), faker.number().numberBetween(1, 10), faker.number().numberBetween(100, 1500)),
-            new Product(faker.medical().medicineName(), faker.number().numberBetween(1, 10), faker.number().numberBetween(100, 1500)),
-            new Product(faker.medical().medicineName(), faker.number().numberBetween(1, 10), faker.number().numberBetween(100, 1500))
-    ));
-
-/*          System.out.println("personMap: " + categoryMap.toString());
-            System.out.println();
-            System.out.println("personMap.keySet(): " + categoryMap.keySet().toString());
+    public Store getRandomStore(int count) {
+        Faker faker = new Faker();
+
+        List<Product> productsOfBooks = new ArrayList<>();
+        List<Product> productsOfBeers = new ArrayList<>();
+        List<Product> productsOfMedicines = new ArrayList<>();
+
+        List<Category> categories = new ArrayList<>();
+
+        Category book = new Book("Book", productsOfBooks);
+        Category beer = new Beer("Beer",  productsOfBeers);
+        Category medicine = new Medicine("Medicine",  productsOfMedicines);
+
+        for (int i = 0; i < count; i++) {
+            productsOfBooks.add(new Product(faker.book().title(), faker.number().numberBetween(1, 10), faker.number().numberBetween(1, 1000)));
+            productsOfBeers.add(new Product(faker.beer().name(), faker.number().numberBetween(1, 10), faker.number().numberBetween(1, 1000)));
+            productsOfMedicines.add(new Product(faker.medical().medicineName(), faker.number().numberBetween(1, 10), faker.number().numberBetween(1, 1000)));
+        }
+
+        categories.add(book);
+        categories.add(beer);
+        categories.add(medicine);
+
+        Store store = new Store(categories);
+
+        System.out.println(store);
+        for (Category category : categories) {
+            System.out.println(category.getCategoryName() + " have");
             System.out.println();
-        for(Category category : categoryMap.keySet()){
-                System.out.println(category.toString() + " имеет");
+            for (Product product : category.getListOfProducts()) {
+                System.out.println("  " + product);
                 System.out.println();
```

Comment:
system out should be moved to separate method. It could be that you need just store and don't want to print it. 
So it better to have printing as separate method